src/typeahead/typeahead.spec.ts

```diff
@@ -513,6 +513,7 @@ describe('ngb-typeahead', () => {
          changeInput(compiled, 'o');
          fixture.detectChanges();
          tick(250);
+         tick(100);  // for the aria-live announcer
```

Comment:
I think we should use a different announcer for tests or inject delay value 0 so it would become synchronous

Reply:
__ANYTHING__ to get rid of these weird fake things like `tick` would please me. I'll explore the injection of a different delay, seems the simplest and most flexible, that could even be useful for users.